torchaudio/datasets/librispeech_finetune.py

```diff
@@ -0,0 +1,100 @@
+import os
+from pathlib import Path
+from typing import Tuple, Union
+
+import torchaudio
+from torch import Tensor
+from torch.hub import download_url_to_file
+from torch.utils.data import Dataset
+from torchaudio.datasets.utils import (
+    extract_archive,
+)
+
+
+FOLDER_IN_ARCHIVE = "librispeech_finetuning"
```

Comment:
Making this `FOLDER_INARCHIVE` global variable, is a practice that does not have clear reasoning / advantage, so can we simply not use this? (same goes to `folder_in_archive` argument in the constructor.)